modules/nf-core/muscle5/super5/tests/main.nf.test

```diff
@@ -26,6 +26,7 @@ nextflow_process {
                 assertAll(
                         { assert process.success },
                         { assert snapshot(process.out.alignment).match("alignment - uncompressed")},
+                        { assert snapshot(process.out.versions).match("versions1") }
```

Comment:
Shouldn't we have just 1 snapshot per test instead?

Reply:
do you have an example of how should it be?

Reply:
done!